### PR TITLE
HAAR-920: Fixing offloc deployment pipeline

### PR DESCRIPTION
**Pull request by brightonsbox**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -18,23 +25,49 @@ jobs:
           key: dependency-cache-{{ checksum "package-lock.json" }}
       - run:
           name: Install Dependencies
-          command: npm ci
+          command: npm ci --no-audit
```
Comment (williamfalconeruk) [inline .circleci/config.yml:28]: are we going to do a separate step to audit the packages? or just a manual thing?

Comment (brightonsbox) [reply .circleci/config.yml:28]: This is done as a nightly job instead (see the hmpps/npm_security_audit job)

Comment (williamfalconeruk) [reply .circleci/config.yml:28]: awesome! got it 👍🏻

**Review by williamfalconeruk — DISMISSED** [review]
:shipit:

**Pull request merged**